owasp-check-suppressions.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+    <suppress>
+        <notes><![CDATA[
+      False positive. No known CVEs in hazelcast-gcp.
```

Comment:
I know this includes some jquery rubbish via hadoop dependencies.

Is it possible to mark as false positive a file in the module rather than whole module? Because if something new comes up and it appears in this module we won't know about it?

Reply:
Or at least ignore the particular vulnerability via `vulnerabilityName` tag.

Reply:
There are more ways how to suppress an issue. Nevertheless,...

I think the findings with transitive vulnerable jQuery are not false positives. Probably we should handle them differently - comment on the report that these vulnerabilities don't affect our usage.